src/Dev/BuildTask.php

```diff
@@ -64,7 +72,12 @@ abstract public function run($request);
      */
     public function isEnabled()
     {
-        return $this->enabled;
+        $isEnabled = $this->config()->get('is_enabled');
+
+        if ($isEnabled === null) {
+            return static::$enabled ?? true;
```

Comment:
```suggestion
            return static::$enabled;
```
As mentioned in another comment we don't need the `?? true` since `static::$enabled` is the fallback

Reply:
I understand what you mean, but it does actually throw errors if I do not add a fallback here. 

Specifically, this error message pops up:

```
[Emergency] Uncaught Error: Access to undeclared static property DevTasks\ExampleTask::$enabled
GET /dev/tasks/
```

This happens with the tasks I wrote myself as well as with third-party tasks from addons. 

I might misunderstand something here, tbh. But I do believe we have to let this in.

Reply:
Oh sorry my bad .. don't use `static::$enabled`, just use `$this->enabled;`

$enabled isn't a static property, it's a regular property

Reply:
Oh yes, that makes an awful lot of sense :D 

Thanks for helping resolve this, I shouldn’t have missed that. My bad!